allennlp/models/esim.py

```diff
@@ -205,8 +205,8 @@ def forward(  # type: ignore
 
         # The pooling layer -- max and avg pooling.
         # (batch_size, model_dim)
-        v_a_max, _ = replace_masked_values(v_ai, premise_mask.unsqueeze(-1), -1e7).max(dim=1)
-        v_b_max, _ = replace_masked_values(v_bi, hypothesis_mask.unsqueeze(-1), -1e7).max(dim=1)
+        v_a_max = masked_max(v_ai, premise_mask.unsqueeze(-1), 1)
```

Comment:
Can you keep the `dim=1` here?  Having a magic value of 1 is not very readable - it leaves me guessing what that 1 means.